rust/Cargo.toml

```diff
@@ -16,13 +16,14 @@ members = ["codegen"]
 prost = { version = "0.10", default-features = false, features = ["prost-derive"] }
 bytes = { version = "1.0.1", default-features = false }
 hex = { version = "0.4.3", default-features = false, features = [ "alloc" ] }
-sha2 = { version = "0.9.3", default-features = false }
-sha3 = { version = "0.9.1", default-features = false }
-ripemd160 = { version = "0.9.1", default-features = false }
 anyhow = { version = "1.0.40", default-features = false }
-sp-std = {version = "3.0.0", default-features = false }
+sp-std = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.22", default-features = false }
```

Comment:
I really don't like tagging to branches here.
Can we just update to a newer release if that is what you need?

Reply:
unfortunately, this won't work. Substrate doesn't publish new releases to crates.io so parachain teams need to track git branches for new releases.

Reply:
why do they do that? do you know?

Reply:
from the versioning (currently we're at 0.9.24) I'd say they don't want to publish until they hit v1.0.0

Reply:
@ethanfrey will you be able to tag a version & [publish on crates.io](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations) with the dependency specified as git/branch?